src/main/java/org/example/AccountInputService.java

```diff
@@ -0,0 +1,65 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class AccountInputService {
+    Scanner sc = new Scanner(System.in);
+    public AccountType getAccountType(){
+        System.out.print("""
+                1. Basic Banking Account
+                 Account can be opened with an initial/minimum deposit of Rs.1, 000.
+                2. Saving Account
+                 Account can be opened with an initial/minimum deposit of Rs.5,000
+                3. Current Account
+                 No Minimum Balance
+                Select Account Type:\s""");
+        int Type = sc.nextInt();
+        AccountType accountType;
+        if(Type==1){
+            accountType = AccountType.BASIC;
+        } else if (Type==2) {
+            accountType = AccountType.SAVING;
+        } else if(Type==3){
+            accountType = AccountType.CURRENT;
+        }else {
+            System.out.println("Invalid account type");
+            return getAccountType();
+        }
+        return accountType;
+    }
+    public int getAccountId(){
+        System.out.print("Enter account ID: ");
+        int accountId = sc.nextInt();
+        return accountId;
+    }
+    public String getAccountPassword(){
+        System.out.print("Enter Password: ");
+        String accountPassword = sc.next();
```

Comment:
add functionality of confirm password, user should enter the confirm password as well